MIP0/mip0.md

```diff
@@ -7,287 +11,254 @@ Title: The Maker Improvement Proposal Framework
 Author(s): Charles St.Louis (@CPSTL), Rune Christensen (@Rune23)
 Contributors: @LongForWisdom
 Type: Process
-Status: Accepted 
+Status: Accepted
 Date Proposed: 2020-04-06
 Date Ratified: 2020-05-02
 Dependencies: n/a
 Replaces: n/a
 ```
 ## References
 **[General-MIP-Template.md](General-MIP-Template.md)**  
-**[Technical-MIP-Template.md](Technical-MIP-Template.md)**  
-**[MIP0c12-Subproposal-Template.md](MIP0c12-Subproposal-Template.md)**  
```

Comment:
We should still have these, imo. The idea of this section is to show everything relevant to the MIP. We'll probably make this section a dropdown or something on the MIPs Portal at some point.